war/src/main/js/plugin-manager-ui.js

```diff
@@ -34,7 +34,7 @@ function applyFilter(searchQuery) {
 
         clearOldResults()
         var rows = pluginManagerAvailable({
-            plugins: plugins.filter(plugin => !selectedPlugins.includes(plugin.name)),
+            plugins: plugins.filter(plugin => selectedPlugins.indexOf(plugin.name) === -1),
```

Comment:
could we just polyfill includes?

Reply:
We should, but for a quick regression fix I'd rather keep it simple and follow up with proper work.

Reply:
@fqueiruga is => working on IE11? I thought it doesn't work https://stackoverflow.com/a/40216100/4728711

Reply:
Yes, it works